.github/workflows/jest-coverage-changes.yml

```diff
@@ -0,0 +1,26 @@
+name: Jest Coverage - changed files
+
+on:
+  pull_request:
+    branches: develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
From what i see in the documentation , `fetch-depth: 0` fetches all the branches and tags. 
https://github.com/actions/checkout/issues/329

```    
# Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
```